pkg/cri/store/image/image.go

```diff
@@ -56,12 +55,6 @@ type Image struct {
 	Pinned bool
 }
 
-// InfoProvider provides both content and info about content
-type InfoProvider interface {
```

Comment:
Do we know if there's external consumers of these interfaces? If so, we should add an alias and deprecate; more important if we decide to backport this patch (to help transition v1 -> v2).

```go
// InfoProvider provides both content and info about content.
//
// Deprecated: use [content.InfoProvider] instead.
type InfoProvider = content.InfoProvider
```

(same for the other ones)

Reply:
`pkg/display` was introduced in v2  and doesn't exist in v1 so we skip the deprecation.

As for the `pkg/cri/store/image` technically it could have external consumers, but looking at https://pkg.go.dev/github.com/containerd/containerd@v1.7.11/pkg/cri/store/image?tab=importedby and https://grep.app/search?q=pkg/cri/store/image it doesn't seem like it has any real publicly known importers.
Also, it's still exactly the same interface, so I think even if there are any consumers, the change should be transparent?